src/com/lftechnology/java/training/srijan/basics/Fabonacci.java

```diff
@@ -0,0 +1,88 @@
+package com.lftechnology.java.training.srijan.basics;
+
+import java.util.logging.Logger;
+
+/**
+ * Each new term in the Fibonacci sequence is generated by adding the previous
+ * two terms. By starting with 1 and 2, the first 10 terms will be: 1, 2, 3, 5,
+ * 8, 13, 21, 34, 55, 89, ... By considering the terms in the Fibonacci sequence
+ * whose values do not exceed four million, find the sum of the even-valued
+ * terms.
+ */
+
+public class Fabonacci {
+	private int firstNumber;
+	private int secondNumber;
+	private final int numberLimit = 4000000;
+	private int[] fabonacciSeries = new int[4000000];
+	private int count = 0;
+	
+	private static final Logger LOGGER = Logger.getLogger(Fabonacci.class.getName());
+
+	public Fabonacci(int firstNumber, int secondNumber) {
+		this.firstNumber = firstNumber;
+		this.secondNumber = secondNumber;
+	}
+
+	public int getFirstNumber() {
+		return firstNumber;
+	}
+
+	public int getSecondNumber() {
+		return secondNumber;
+	}
+
+	public void setFirstNumber(int firstNumber) {
+		this.firstNumber = firstNumber;
+	}
+
+	public void setSecondNumber(int secondNumber) {
+		this.secondNumber = secondNumber;
+	}
+
+	/**
+	 * <p>
+	 * Display all Fabonacci Number between 1 and 4 million
+	 * </p>
+	 * @author Srijan Bajracharya <srijanbajracharya@lftechnology.com>
+	 */
+
+	public void displayFabonacciNumber() {
+		int sum = 0;
+		do {
+
+			sum = this.firstNumber + this.secondNumber;
+			this.firstNumber = this.secondNumber;
+			this.secondNumber = sum;
+			if (sum < 4000000) {
+				count++;
+				LOGGER.info(" " + sum + " " );
+				fabonacciSeries[count] = sum;
+			}
+
+		} while (sum < numberLimit);
+	}
+
+	/**
+	 * <p>
+	 * Display Even Fabonacci Series
+	 * </p>
+	 * <p>
+	 * Calculates and display sum of Even fabonacci series
+	 * </p>
+	 * @author Srijan Bajracharya <srijanbajracharya@lftechnology.com>
+	 */
+
+	public void displaySumOfEvenFabonacciNumber() {
```

Comment:
Codes are not clean. Consider the following it will help you:
use Ctrl+Shift+F to format your code
Remove unused new lines. 
Do not add space after method/class documentation